src/turbomind/models/llama/LlamaV2.cc

```diff
@@ -131,6 +133,7 @@ void LlamaV2<T>::initialize(const LlamaAttentionParams& attn_params,
                                                  stream_,
                                                  cublas_wrapper_,
                                                  allocator_,
+                                                 lora_params_,
```

Comment:
`lora_param_` is passed through all abstration layers. And the checking of `lora_param_.policy == 1` does not seem necessary.